include/README

```diff
@@ -0,0 +1,76 @@
+ODP specification
+=================
+
+ODP specification consists of several types of files, which together provide
+full list of types, values and functions that ODP implemention MUST provide.
+
+ODP API specification
+---------------------
+
+These are the files from `include/odp/api/spec` directory. They specify a set
+of functions that MUST be provided by ODP implementation. Doxygen comments
+inside these files document requirements for ABI interface provided by an
+implementation. An implementation MUST use these headers AS IS, without any
+modifications to be compatible with ODP specification.
+
+ODP ABI compatibility specification
+-----------------------------------
+
+These are the files from `include/odp/arch/@ARCH_ABI@/odp/api/abi/` directory.
+They specify a set of types and values that MUST be used AS IS without any
+modifications by an implementation if it supports and is compiled for
+ABI-compatibility mode.
+
+ODP default ABI headers
+-----------------------
+
+These are the files from `include/odp/api/abi-default` directory. They provide
+default specification for ODP types and values. ODP ABI compatibility files
+heavily depend on these headers. These files MUST NOT be changed by an
+implementation. If an implementation is compiled for ABI-compatibility mode,
+these headers should be used AS IS.
+
+Additional ODP headers
+======================
+
+These are the headers provided by an ODP to supplement ODP specification.
+
+ODP API headers
+---------------
+
+These are the files from `include/odp/api` directory. They glue together API
+and ABI specification headers. Although they are not part of ODP specification
+itself, they provide an easy way for an application to use ODP. An
+implementation SHOULD use these headers AS IS unless it has strong reason not
+to do so.
+
+odp_api.h header
+----------------
+
+This header found at `include/odp_api.h` is a main entry point for an
+application. It includes all ODP API headers, so that an application doesn't 
+have to include them one by one. It is not a part of ODP specification
+
```

Comment:
Application MUST include only odp_api.h, nothing else. So, the file existence is part of ODP spec, the content is not other than the entire API spec is visible through it.